test/ehttpc_tests.erl

```diff
@@ -95,7 +95,7 @@ kill_gun_resume_test() ->
 send_100_test_() ->
     Port = ?PORT,
     ServerOpts1 = #{port => Port, name => ?FUNCTION_NAME, delay => 3, oneoff => true},
-    ServerOpts2 = #{port => Port, name => ?FUNCTION_NAME, delay => {rand, 300}, oneoff => false},
+    ServerOpts2 = #{port => Port, name => ?FUNCTION_NAME, delay => {rand, 3}, oneoff => false},
```

Comment:
Q: would it be better to keep 300 to have a bit more room for variability in the behaviors?

Reply:
Then the test case will fail due to timeout. Before my fix of the test server, there was no delay between most requests but after my fix there will be a delay between every request. Therefore with the same delay config the test takes much longer with the fixed test server.